src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -61,16 +67,18 @@ export const MapRoom: React.FC<MapRoomProps> = ({
     <div
       className={containerClasses}
       style={roomPositionStyles}
-      onClick={selectRoom}
-      onMouseEnter={handleRoomHovered}
-      onMouseLeave={handleRoomUnhovered}
+      onClick={isUnclickable ? noop : selectRoom}
+      onMouseEnter={isUnclickable ? noop : handleRoomHovered}
+      onMouseLeave={isUnclickable ? noop : handleRoomUnhovered}
```

Comment:
We could handle this in 'one hit' by spreading these props in only when relevant.

Reply:
This works though.. going to call this wontfix.